### PR TITLE
chore: fix pre-commit eslint dependencies

### PR DESCRIPTION
**Pull request by ssbarnea**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -171,6 +171,7 @@
     "@commitlint/cli": "latest",
     "@commitlint/config-conventional": "latest",
     "husky": "latest",
+    "typescript-eslint": "^0.0.1-alpha.0",
```
Comment (joshuawilson) [inline package.json:174]: Why is this added to the Dependencies?

Comment (ssbarnea) [reply package.json:174]: That is indeed a mistake, I am still trying to make pre-commit.ci happy....
